packages/widget_toolkit_otp/lib/src/base/extensions/error_model_extensions.dart

```diff
@@ -0,0 +1,11 @@
+import '../models/error/error_model.dart';
+
+extension ExceptionToErrorModel on Exception {
+  ErrorModel asErrorModel() =>
+      this is ErrorModel ? this as ErrorModel : ErrorUnknown(exception: this);
```

Comment:
We need to find a better way of handling the errors across the app and the packahe